src/textual/reactive.py

```diff
@@ -150,7 +151,7 @@ def __rich_repr__(self) -> rich.repr.Result:
         yield "name", getattr(self, "name", None), None
 
     @classmethod
-    def _clear_watchers(cls, obj: Reactable) -> None:
+    def _clear_watchers(cls, obj: MessagePump) -> None:
```

Comment:
Given that a Reactable is a DOMNode, why is this change neccesary?

Reply:
_clear_watchers is called with a less specific type than Reactable or DOMNode elsewhere